torch/_dynamo/output_graph.py

```diff
@@ -232,6 +232,11 @@ def __init__(
         self.export_constraints = export_constraints
         self.frame_state = frame_state
         self.tensor_weakref_to_sizes_strides: WeakIdKeyDictionary = {}
+
+        # Used to maintain an alias between real values variable tracker for tensors we have seen.
+        # This map ensures that the only tensors in graph inputs, and the only tensors in guards are unique.
+        self.real_value_tensor_positive_aliases = WeakTensorKeyDictionary()
```

Comment:
@voznesenskym where is this map populated?
At the moment, in master, we are checking whether something is in this variable before using it, so is it being used at all?